packages/web/src/pages/sign-up-page/components/ContinueFooter.module.css

```diff
@@ -0,0 +1,6 @@
+.container {
+  box-shadow: var(--harmony-shadow-mid-inverted);
+  width: 100%;
```

Comment:
Add `w='100%' shadow='midInverted'` to the Flex

Reply:
omg nice

Reply:
there's also an emotion css prop if you wanna pass in the non-standard stuff and have no css file

Reply:
ahh ok good to know!